Sources/swift-algorand-sdk/model/TransactionParametersResponse.swift

```diff
@@ -7,45 +7,55 @@
 
 import Foundation
 
-public class TransactionParametersResponse:Codable {
+public struct TransactionParametersResponse:Codable, Equatable {
+
+    
     public  var consensusVersion:String?;
     public  var fee:Int64?;
-    public  var genesisHash:[Int8]?;
     public var genesisId:String?;
     public    var lastRound:Int64?;
     public   var minFee:Int64?;
 
-    
+    private var gHash64:String?
+    public var genesisHash:[Int8]? {
+        get {
+            if let gHash64 = gHash64 {
+                return CustomEncoder.convertToInt8Array(input: CustomEncoder.decodeByteFromBase64(string: gHash64))
+            }
+            return nil
+        }
+    }
```

Comment:
@Jesulonimi21 can you double-check i didn't mess everything up here? :D
The main idea is to remove custom decoding phase (otherwise the "encode" counterpart needed for tests fails, as they are not symmetrical for the genesisHash property).

This is however under unit test at the moment, and is passing, just wanted to have a double check I properly understood the custom encoding.